### PR TITLE
Extend setup tdcs

### PR DESCRIPTION
**Pull request by moritzdannhauer**
@dcwhite the function create_rhs does generate valid output return matrix but its all nan in algo's run function. any ideas why that is? thanks!

### CODE REVIEW

**Comment by dcwhite** [comment]
@moritzdannhauer Careful, you deleted the file bin/.gitignore which causes Travis to fail and all sorts of other badness. Please resurrect it.

**Comment by moritzdannhauer** [comment]
ok, i just force-add it to the branch.

**Comment by moritzdannhauer** [comment]
@dcwhite : It should be ready for you to review and merge to master.

File: src/ExampleNets/regression/setuptdcs_colin27_patchelc.srn5 [hunk 1]
```diff
@@ -0,0 +1,1614 @@
...
+							<item>
+								<first>
+									<name>Filename</name>
+								</first>
+								<second>
+									<name>Filename</name>
+									<value>
+										<which>2</which>
+										<value>%SCIRUNDATADIR%/Fields/setuptdcs/example/cloin27_skin_surf.fld</value>
+									</value>
```
Comment (dcwhite) [inline src/ExampleNets/regression/setuptdcs_colin27_patchelc.srn5]: @moritzdannhauer Typo in these filenames: "cloin" should be "colin"

**Pull request merged**